app/views/groups/_form.html.erb

```diff
@@ -1,24 +1,24 @@
+<%= render partial: 'shared/nav', locals: {title:'CATEGORY', symbol:'←', url: user_groups_path}%>
+
 <%= form_with model: group, url: user_groups_path, local: true, multipart: true  do |form| %>  
-  <div class="post_form">
-    <h3> Add New Category </h3>
-      <div>
```

Comment:
### Add a new transaction page

- [ ]  According to the project requirements, When the user clicks on the "Back" button (<), the user navigates to the transactions page for that category and not the Home Page. Kindly correct this as well. ✍🏻